src/viur_cli/conf.py

```diff
@@ -86,7 +87,34 @@ def delete(self):
         except:
             raise click.ClickException(click.style(f"{configname} not found", fg="red"))
 
+    def find_key(self, dictionary, target_key, target, level=0):
+        if target_key in dictionary:
+            value = dictionary[target_key]
+
+            if target is None:
+                self[target_key] = value
+                del value
+                return
+            else:
+                self[target][target_key] = value
+                return
```

Comment:
1. There is no need for the `del`
2. Return in both if and else case is awkward
3. What is the sense of the `if`-case when the target is None, and None is assigned to `self[target_key]`?